subsys/shell/Kconfig

```diff
@@ -111,8 +111,15 @@ config SHELL_START_OBSCURED
 	  If enabled, don't echo actual character, but echo * instead.
 	  This is used for login prompts.
 
+config SHELL_VT100_COMMANDS
+	bool "Enable VT100 commands in shell"
+	default y if !SHELL_MINIMAL
+	help
+	  Enables VT100 commands in shell (e.g. cursor position, clear screen etc.).
+
 config SHELL_VT100_COLORS
 	bool "Enable colors in shell"
+	depends on SHELL_VT100_COMMANDS
```

Comment:
I'm not convinced that we want to deprive Shell's minimal version of VT100 support. Apart from that I am fine with the change.